version_matrix.yaml

```diff
@@ -0,0 +1,57 @@
+git_hash: <git_hash_here>
```

Comment:
I don't think I understand what this file is for, if it's got a git_hash embedded into it?